server/routes/getValidatorDetail.ts

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable guard-for-in */
+import { Request, Response, NextFunction } from 'express';
+import { factory, formatFilename } from '../../shared/logging';
+import { Op } from 'sequelize';
+function flatten(validators) {
+    validators = JSON.parse(JSON.stringify(validators));
+    const filteredValidators = validators.rows.map((row) => {
+        let dataObject = { ...row, ...row.Validator };
+        delete dataObject.Validator;
+        return dataObject;
+    });
+    return filteredValidators;
+}
+
+function formatData(searchCriteria) {
+    for (let key in searchCriteria) {
+        if (key === 'name' || key === 'stash_id') {
+            let temp = { ...searchCriteria };
+            delete searchCriteria[key];
+            searchCriteria = {
+                ...searchCriteria,
+                [key]: {
+
+                    [Op.iLike]: `%${temp[key]}%`
+                }
+            }
+        }
+    }
+    return searchCriteria;
+}
+export const getCurrentValidators = async (models, req: Request, res: Response, next: NextFunction) => {
+    let currentValidators: any = [];
+    let { pagination = { currentPageNo: 1, pageSize: 20 }, searchCriteria } = req.query;
+    searchCriteria = formatData(searchCriteria);
+    currentValidators = await models.HistoricalValidatorStats.findAndCountAll({
```

Comment:
getting current validators should come from the validators table not the historical statistics table. Historical statistics table is explicitly for historical data, not active data.